.github/workflows/selenium.yaml

```diff
@@ -0,0 +1,43 @@
+name: Selenium tests
+on: [push, pull_request]
+env:
+  GALAXY_TEST_DBURI: 'postgres://postgres:postgres@localhost:5432/galaxy?client_encoding=utf8'
+  GALAXY_TEST_SKIP_FLAKEY_TESTS_ON_ERROR: 'true'
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7']
+        chunk: [0, 1, 2]
+    services:
+      postgres:
+        image: postgres:11
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: 'galaxy root'
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Cache pip dir
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: pip-cache-${{ matrix.python-version }}-${{ hashFiles('galaxy root/requirements.txt') }}
+      - uses: nanasess/setup-chromedriver@master
+      - name: Run tests
+        run: ./run_tests.sh -selenium lib/galaxy_test/selenium -- --num-shards=3 --shard-id=${{ matrix.chunk }}
+        working-directory: 'galaxy root'
+      - uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: Selenium test results
+          path: 'galaxy root/database/test_errors'
```

Comment:
We're losing all the screenshots for the passing PRs right? I put a lot of work into those and I really think they are valuable, but the cost of all the noise and having to maintain a buggy Jenkins setup is probably too high.

Reply:
Yes, I think as an artefact that would probably push us to the limit of what we can store. There is always an option to upload to S3 or similar, if we can get a bit of S3 storage.